### ISSUE TITLE
Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an al...

### ISSUE THREAD

**Issue by int32at** [sgengler/atom-trello#1]
[Enter steps to reproduce below:]
1. Add following JS code to the description of a card (incl markdown)

`````` js
<script type="text/javascript"><!--//<![CDATA[
var EM_reListId = 255;
var EM_del = 0;
var EM_protocol = (location.protocol=="https:"?"https://":"http://");
var EM_timestamp = Math.floor(Math.random()*99999999999);
  document.write ("<scr"+"ipt type='text/javascript' src='"+EM_protocol);
  document.write ("www.rvty.net/ads/ReTgt.js?timestamp="+EM_timestamp);
  document.write ("'><\/scr"+"ipt>");
//]]>--></script>```
2. Ttry to open the card within Atom

hope this helps. let me know if you need more info :)

**Atom Version**: 1.0.18
**System**: Mac OS X 10.10.5
**Thrown From**: [atom-trello](https://github.com/sgengler/atom-trello) package, v0.1.1


### Stack Trace

Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".


``````

At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:328

EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

```
at eval (native)
at Function.jQuery.extend.globalEval (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:328:5)
at [object Object].jQuery.fn.extend.domManip (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:5435:16)
at [object Object].jQuery.fn.extend.append (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:5218:15)
at AtomTrelloView.module.exports.SelectListView.populateList (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:208:21)
at AtomTrelloView.module.exports.SelectListView.setItems (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:167:12)
at AtomTrelloView.module.exports.AtomTrelloView.showView (/Users/andreas/.atom/packages/atom-trello/lib/atom-trello-view.coffee:64:6)
at /Users/andreas/.atom/packages/atom-trello/lib/atom-trello-view.coffee:121:8
at Request._callback (/Users/andreas/.atom/packages/atom-trello/node_modules/node-trello/lib/node-trello.js:94:5)
at Request.self.callback (/Users/andreas/.atom/packages/atom-trello/node_modules/node-trello/node_modules/request/request.js:373:22)
```

```

### Commands

```

  2x -0:09.1.0 core:close (div.release-notes.padded.pane-item.native-key-bindings)
     -0:07.9.0 command-palette:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-base16-tomorrow-dark-theme.theme-isotope-ui)
     -0:06.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:06.5.0 atom-trello:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-base16-tomorrow-dark-theme.theme-isotope-ui)
     -0:04.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)

``````

### Config

```json
{
  "core": {
    "themes": [
      "isotope-ui",
      "base16-tomorrow-dark-theme"
    ]
  },
  "atom-trello": {
    "devKey": "xxx",
    "token": "xxx"
  }
}
``````
### Installed Packages

``` coffee
# User
atom-html-preview, v0.1.11
atom-trello, v0.1.1
autocomplete-python, v0.5.4
isotope-ui, v2.5.1
terminal-panel, v1.14.1

# Dev
No dev packages
```

**Comment by sgengler** [comment]
Thanks for sending. I just published an update (1c3bbb3a9bba2d0b26a4026e01173547f37e0ddf) which sanitizes the titles and parses any markdown in the description. Should be all set now.
<img width="547" alt="screen shot 2015-09-25 at 1 20 24 pm" src="https://cloud.githubusercontent.com/assets/3884266/10107484/6b6e767c-6389-11e5-8622-d6c3337e3b59.png">

**Comment by int32at** [comment]
awesome, thank you so much :) :+1:
